xonsh/main.py

```diff
@@ -128,43 +128,41 @@ def parser():
         dest="help",
         action="store_true",
         default=False,
-        help="show help and exit",
+        help="Show help and exit.",
     )
     p.add_argument(
         "-V",
         "--version",
-        dest="version",
-        action="store_true",
-        default=False,
-        help="show version information and exit",
+        action="version",
+        help="Show version information and exit.",
+        version="/".join(("xonsh", __version__)),
```

Comment:
I think f"xonsh/{__version__}" would be better here as the value

Reply:
Yeah, it looks better! Changed.